include/xtensor/xreducer.hpp

```diff
@@ -525,6 +525,24 @@ namespace xt
         return reducer_type(std::forward<RF>(reduce_func), std::forward<IF>(init_func), std::forward<MF>(merge_func));
     }
 
+    template <std::size_t I, typename ...T>
+    decltype(auto) get(xreducer_functors<T...>&& v)
```

Comment:
These functions can be removed when the generic versions have been implemented in `xutils.hpp`.

Reply:
Done.